### DISCUSSION TITLE
Solidity

### DISCUSSION THREAD

**Discussion by shlokvarma45** [Cyfrin/foundry-full-course-cu#2729] [Q&A]
Hello fellow devs, can I know what are few websites that provide gigs, projects and bounties for solidity developers? Any leads appreciated

**Comment by EngrPips** [comment]
Hello @shlokvarma45, I don't know of any but I will be glad to know what you find out.

**Comment by chainflash** [comment]
https://gitcoin.co
https://bounties.network
you can check these two sites

**Reply by EngrPips** [reply]
Thanks for the heads-up
